documentation/contributing/introduction.adoc

```diff
@@ -6,20 +6,29 @@ Strimzi documentation is written in Asciidoc.
 
 == Documentation tools
 
+Following tools are needed to build the documentation:
+
 https://asciidoctor.org/[`asciidoctor`^]:: Documentation generation tool
-https://www.gnu.org/software/make/[`make`^]:: Make build system to build complete documentation including API reference
+https://www.gnu.org/software/make/[`make`^]:: Make build system to build complete documentation
 https://github.com/mikefarah/yq[`yq`^]:: YAML build tool to build documentation using _make_ targets
 
+Additionally, for generating the API reference you will also need:
+
+https://adoptopenjdk.net/[`java`^]:: Java for building code and generating the API reference
+https://maven.apache.org/[`mvn`^]:: Maven for running the Java build
+
 For most documentation updates, `asciidoctor` offers the simplest way to check the build.
 
 == Project layout
 
 Strimzi uses git to manage repositories.
 
-https://github.com/strimzi/strimzi-kafka-operator[`strimzi-kafka-operator`^] (GitHub):: The public GitHub repo hosts all of the Strimzi code and documentation.
+https://github.com/strimzi/strimzi-kafka-operator[`strimzi-kafka-operator`^] (GitHub):: The public GitHub repo hosts all of the Strimzi Operators code and documentation.
 https://github.com/strimzi/strimzi-kafka-bridge[`strimzi-kafka-bridge`^] (GitHub):: The public GitHub repo hosts all of the Strimzi Kafka Bridge code.
 https://github.com/strimzi/strimzi-kafka-oauth[`strimzi-kafka-oauth`^] (GitHub):: The public GitHub repo hosts all of the Strimzi OAuth 2.0 code.
-https://github.com/strimzi/strimzi-kafka-operator/tree/master/documentation[`strimzi-kafka-operator/documentation`^] (GitHub):: The _documentation_ folder is split into _category_ folders to manage the content.
+
+Most of the documentation is in the Strimzi Operators repository, in the https://github.com/strimzi/strimzi-kafka-operator/tree/master/documentation[`documentation` folder^].
```

Comment:
Most? Where else?

Reply:
The Bridge documentation is generated from the OpenAPI spec in the Bridge repo. TBH I didn't want to get lost into explaining the details of that. But I guess this might look a bit weird. Should I remove the _Most of ..._ or should I try to explain the details of the OpenAPI spec? WDYT?